tests/cpp-tests/Classes/UITest/CocoStudioGUITest/UIScrollViewTest/UIScrollViewTest_Editor.cpp

```diff
@@ -163,3 +164,30 @@ bool UIScrollViewTest_ScrollToPercentBothDirection_Bounce_Editor::init()
     
     return false;
 }
+
+// UIScrollViewTest_Vertical_Editor
+
+UIScrollViewTest_ClippingWithNode::UIScrollViewTest_ClippingWithNode()
+{
+
+}
+
+UIScrollViewTest_ClippingWithNode::~UIScrollViewTest_ClippingWithNode()
+{
+
+}
+
+bool UIScrollViewTest_ClippingWithNode::init()
+{
+    if (UIScene_Editor::init())
+    {
+        FileUtils::getInstance()->addSearchPath("ccs-res/hd/cocosui/UIEditorTest/UIScrollView/ScrollViewAndButton/");
```

Comment:
`ccs-res/hd/cocosui/UIEditorTest/UIScrollView/ScrollViewAndButton/` should be `cocosui/UIEditorTest/UIScrollView/ScrollViewAndButton/`